src/desktop/components/search_bar/collections/search.coffee

```diff
@@ -23,21 +26,34 @@ module.exports = class Search
     "?#{@data()}"
 
   data: ->
-    data = visible_to_public: true, fair_id: @fairId, size: @size
+    data = visible_to_public: true, fair_id: @fairId, size: @size, agg: @agg
     "#{qs.stringify data}&term=%QUERY" # Requires an unencoded percent character
 
-  parseResults: (items) ->
-    if @restrictType?
-      @restrictType = [@restrictType] unless _.isArray @restrictType
-      _.filter items, (item) =>
-        _.contains @restrictType, item?.owner_type
-    else
-      items
-
-  parse: (items, query) ->
-    # HACK filter out sensitive results
-    items = _.reject items, (item) ->
+  filterResults: (items) ->
+    _.reject items, (item) ->
       JSON.stringify(item).match(/kippenberger|zoe.*leonard/i)
-    @results.reset _.map @parseResults(items), (item) =>
+
+  parseResults: (items, aggregations) ->
+    if aggregations
+      results = []
+      Object.keys(items).forEach((key) ->
```

Comment:
This doesn't put the groupings in any particular order, if we wanted to always have 'Artists', then 'Sales', etc. we could easily do that here.

Reply:
Yes we need to enforce the order outlined here: https://www.notion.so/artsy/Discover-c25acaaf8e6d4b9480a66826bd521bf1?p=e7dd56c8ccb84b05a4f048202a2dc750 

Shows are not in auto-suggest you can ignore that type.

Reply:
Cool, wasn't sure from that doc if the order was decided or not. I'll add the ordering, and thank you for the gallery/institution breakdown as well!